autoware_launch/launch/planning_simulator.launch.xml

```diff
@@ -65,6 +65,7 @@
       <arg name="lateral_controller_mode" value="mpc_follower" />
       <!-- options for longitudinal_controller_mode: pid -->
       <arg name="longitudinal_controller_mode" value="pid" />
+      <arg name="use_external_emergency_stop" value="false" />
```

Comment:
It would be better to add comments for this args same as other args

Reply:
I added comment.